scripts/pod_lib_lint.rb

```diff
@@ -42,13 +48,28 @@ def main(args)
 
   command = %w(bundle exec pod lib lint --sources=https://cdn.cocoapods.org/)
 
+  # Split arguments that need to be processed by the script itself and passed
+  # to the pod command.
+  script_args = {}
+  pod_args = []
+  local_specs_to_ignore_arg = ""
+  args.each do |arg|
+    case arg
+    when /--ignore-local-specs=*/
+      then local_specs_to_ignore_arg = arg.split("=")[1]
+    else
+      pod_args.append arg
+    end
+  end
+
   # Figure out which dependencies are local
   podspec_file = args[0]
```

Comment:
Doesn't this need to refer to `pod_args[0]`? Otherwise if `args[0]` is `--ignore-local-specs` doesn't this end up broken?

Reply:
Updated to use `pod_args[0]`. It still may fail though e.g. when the first arg is `--platforms=`, etc. I may add parsing of the `podspec_file` with a regex to catch this case, though, I it looks fine to me if we just keep in mind that the podspec must be the first argument.